BungeeCord-Patches/0064-Adventure.patch

```diff
@@ -0,0 +1,1874 @@
+From d5c909341edc81b0a3417e989a2a6253c5f7f0ab Mon Sep 17 00:00:00 2001
+From: Callum Seabrook <callum.seabrook@prevarinite.com>
+Date: Wed, 28 Apr 2021 17:02:18 +0100
+Subject: [PATCH] Adventure
+
+
+diff --git a/api/pom.xml b/api/pom.xml
+index ca1ed881..215f89bb 100644
+--- a/api/pom.xml
++++ b/api/pom.xml
+@@ -18,6 +18,28 @@
+     <name>Waterfall-API</name>
+     <description>API implemented by the Elastic Portal Suite</description>
+ 
++    <!-- Waterfall start - add snapshot repository for Adventure's BungeeCord serializer and add BOM -->
++    <repositories>
++        <!-- Waterfall - Add repository required for Adventure's BungeeCord serializer -->
++        <repository>
++            <id>sonatype-oss-snapshots</id>
++            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
++        </repository>
++    </repositories>
++
++    <dependencyManagement>
++        <dependencies>
++            <dependency>
++                <groupId>net.kyori</groupId>
++                <artifactId>adventure-bom</artifactId>
++                <version>4.7.0</version>
++                <type>pom</type>
++                <scope>import</scope>
++            </dependency>
++        </dependencies>
++    </dependencyManagement>
++    <!-- Waterfall end -->
++
+     <dependencies>
+         <dependency>
+             <groupId>io.github.waterfallmc</groupId>
+@@ -82,5 +104,23 @@
+             <artifactId>slf4j-api</artifactId>
+             <version>1.7.25</version>
+         </dependency>
++        <!-- Waterfall - Add Adventure -->
++        <dependency>
++            <groupId>net.kyori</groupId>
++            <artifactId>adventure-api</artifactId>
++        </dependency>
++        <dependency>
++            <groupId>net.kyori</groupId>
++            <artifactId>adventure-text-serializer-gson</artifactId>
++        </dependency>
++        <dependency>
++            <groupId>net.kyori</groupId>
++            <artifactId>adventure-text-serializer-legacy</artifactId>
++        </dependency>
++        <dependency>
++            <groupId>net.kyori</groupId>
++            <artifactId>adventure-text-serializer-bungeecord</artifactId>
++            <version>4.0.0-SNAPSHOT</version>
++        </dependency>
```

Comment:
missing end comment

Reply:
Yeah I saw that when manually checking my patch. Got a bit scared that manually changing it could break it, so decided against that